blog-service/src/main/java/com/in2it/blogservice/controller/BlogServiceController.java

```diff
@@ -128,8 +129,13 @@ public ResponseEntity<ResponseHandler<Boolean>> deleteBlogBytitle(@PathVariable
 	@Operation(summary = "Get a all Blogs", description = "Returns  all blogs  ")
 	public ResponseEntity<ResponseHandler<List<BlogDto>>> getAllBlog(@RequestParam(defaultValue = "0") int pageNum,
 			@RequestParam(defaultValue = "5") int pageSize) {
```

Comment:
why default 5 , not 10

Reply:
By default it was taken for testing purpose because frontend team don't give any specific value .
Second reason is we don't have multiple data on database